### PR TITLE
Main

### PR DESCRIPTION
**Pull request by arimateia98**
**What is the purpose of this pull request?**
The purpose of this pull request its to have a safer way to read data from a json, and a easy way to run the project.

**What problem is this solving?**
Its solving two problems:
 1. instead of running a lot of commands, now it just will need to run npm start.
 2. before this pr, if the file array.json was missing, the program was going to crash, now this is fixed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,12 +4,17 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "tsc src/Ser.ts && tsc src/ControllerSer.ts && tsc src/Main.ts && node src/Main.js",
```
Comment (FMota0) [inline package.json]: `"start": "ts-node src/Main.ts"`

File: package.json [hunk 2]
```diff
@@ -4,12 +4,17 @@
...
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "devDependencies": {
     "@types/node": "^10.12.18"
+  },
+  "dependencies": {
+    "ramda": "^0.26.1",
```
Comment (FMota0) [inline package.json:18]: `tsc` should be on dependencies

**Review by FMota0 — APPROVED** [review]
Seems good

File: package.json (hunk 1 above)
Comment (arimateia98) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (arimateia98) [reply package.json:18]: done

**Review by arimateia98 — COMMENTED** [review]
.

**Review by FMota0 — APPROVED** [review]
Seems good

**Pull request merged**